tests/Custom/CapabilityTest.php

```diff
@@ -14,9 +17,26 @@ class CapabilityTest extends \PHPUnit\Framework\TestCase
      */
     public function testThatCapabilitiesAreAsExpected(string $browserName, array $desiredCapabilities, array $expectedCapabilities): void
     {
-        $driver = $this->createDriverExposingCapabilities($browserName, $desiredCapabilities);
+        $mockWebDriver = $this->createMock(RemoteWebDriver::class);
+        $mockWebDriverOptions = $this->createMock(WebDriverOptions::class);
+        $mockWebDriverTimeouts = $this->createMock(WebDriverTimeouts::class);
+        $mockWebDriver->method('manage')->willReturn($mockWebDriverOptions);
+        $mockWebDriverOptions->method('timeouts')->willReturn($mockWebDriverTimeouts);
```

Comment:
Rearranging this code like so would make it easier to read (same concept as in the https://github.com/minkphp/webdriver-classic-driver/pull/58/files#r1903125007):

```php
$mockWebDriverTimeouts = $this->createMock(WebDriverTimeouts::class);
$mockWebDriverOptions = $this->createMock(WebDriverOptions::class);
$mockWebDriverOptions->method('timeouts')->willReturn($mockWebDriverTimeouts);

$mockWebDriver = $this->createMock(RemoteWebDriver::class);
$mockWebDriver->method('manage')->willReturn($mockWebDriverOptions);
```

P.S.
The mocking code looks very similar to #58. Maybe it can be extracted to a new method used in both PR's later one (once they're merged).

Reply:
done, and moved it into trait, for the other case and for later.